src/Scene.cs

```diff
@@ -72,13 +75,13 @@ public bool Calculate()
                 _levelsToKill = DetectAndMarkLevelsToKill(_ground); // Mark levels that need to be killed...
 
                 if (!_levelsToKill.Any()) // ...if would found so, world will be on pause for 1 iteration.
-                {
+                {                         //   ****can't see where is code to pause the world
```

Comment:
Is that a question OR something you would like to see in the public code base? :) As for an answer, can't give it right away since I didn't touch it for months.